### PR TITLE
add readme note about certbot plugin

### PR DESCRIPTION
**Pull request by kwypchlo**
improve in-code documentation about certbot plugin

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -41,6 +41,10 @@ services:
       - 9980
 
   certbot:
+    # replace this image with the image supporting your dns provider from
+    # https://hub.docker.com/r/certbot/certbot and adjust CERTBOT_ARGS env variable
+    # note: you will need to authenticate your dns request so consult the plugin docs
+    # configuration https://eff-certbot.readthedocs.io/en/stable/using.html#dns-plugins
```
[the remark this answers is not in the document]
Comment (meeh0w) [reply docker-compose.yml:47]: I guess this is what this sentence is about:
>  note: you will need to authenticate your dns request so consult the plugin docs configuration

But at the same time it looks like credentials file is a requirement for most of the [plugins](https://eff-certbot.readthedocs.io/en/stable/using.html#dns-plugins). Maybe we should add a little note on how to mount such a file for the docker container.

[the remark this answers is not in the document]
Comment (meeh0w) [reply docker-compose.yml:47]: I guess this is what this sentence is about:
>  note: you will need to authenticate your dns request so consult the plugin docs configuration

But at the same time it looks like credentials file is a requirement for most of the [plugins](https://eff-certbot.readthedocs.io/en/stable/using.html#dns-plugins). Maybe we should add a little note on how to mount such a file for the docker container.

[the remark this answers is not in the document]
Comment (ro-tex) [reply docker-compose.yml:47]: Yes, one short example goes a long way. That's why I left my setup here.

[the remark this answers is not in the document]
Comment (kwypchlo) [reply docker-compose.yml:47]: Added Cloudflare setup example based on your comments https://github.com/SkynetLabs/skynet-webportal/pull/2164/commits/f3ec6f5f33218055b7b2c1d4a74e14b73a79ff22

**Review by meeh0w — APPROVED** [review]

**Review by ro-tex — APPROVED** [review]

**Pull request merged**